tests/Helper/Log/GoogleApiClientTest.php

```diff
@@ -99,6 +101,8 @@ public function testPostDataWithLatestDrupalVersion(): void {
       'sut' => 'drupal/example',
       'status' => 'PASS',
       'version' => DrupalCoreVersionEnum::CURRENT(),
+      'allowedToFail' => TRUE,
```

Comment:
Should there be a test case somewhere for `'allowedToFail' => FALSE`?

Reply:
No, I don't think so, we are just testing data transfer here, what value we are sending that won't impact any parts of ORCA.